securedrop-workstation-svs-disp/debian/securedrop-workstation-svs-disp.install

```diff
@@ -1,2 +1 @@
-mimeapps.list usr/share/applications/
 paxctld.conf.svsdisp opt/
```

Comment:
the pax flags are also handled by the config package[1] , we can also remove this and relevant postinst chances  

Given that this package will not be installed in consolidated template, we don't strictly need to add this here, we can just remove all the `workstation-svs-disp` related stuff after we release consolidated templates. 


[1] https://github.com/freedomofpress/securedrop-debian-packaging/pull/190/files

Reply:
> we can just remove all the workstation-svs-disp related stuff after we release consolidated templates.

Right, that was my plan: we'll circle back and delete all these packages once the release has shipped. I don't see a pressing need to do that right now.